frontend/app/src/components/home/profile/ReferUsers.svelte

```diff
@@ -53,6 +53,15 @@
             dispatch("chatWith", viewedUserId);
         }
     }
+
+    function buildUsername(userStore: UserLookup, userId: string): string {
```

Comment:
Don't we do this exact same thing in a few places already? It seems like we should be using a shared function for this.